Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -107,6 +97,28 @@ public extension MoyaProvider {
     // swiftlint:enable cyclomatic_complexity
     // swiftlint:enable function_body_length
 
+    private func unstubbedRequest(_ target: Target, request: URLRequest, callbackQueue: DispatchQueue?, progress: Moya.ProgressBlock?, completion: @escaping Moya.Completion) -> Cancellable {
```

Comment:
Sure, sounds good, better than what it is now. :) I've named the method `performRequest`. Hope it's more readable now.